android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -134,12 +134,13 @@ class ModelViewer : android.view.View.OnTouchListener {
     }
 
     @Suppress("unused")
-    constructor(textureView: TextureView) {
+    constructor(textureView: TextureView, engine: Engine = Engine.create()) : this(engine) {
         cameraManipulator = Manipulator.Builder()
                 .targetPosition(0.0f, 0.0f, -4.0f)
```

Comment:
The camera is always positioned at z=-4, should we make that an option too? Or should the manipulator be passed by the client? That way they could use a free fly camera if they so desire.